### PR TITLE
API_URL / WEBAPP_URL defined in env files

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
In https://github.com/airbytehq/airbyte/pull/2706, I introduced a new `WEBAPP_URL` in the envConfigs because I need to include a link to the connection / log page when a job notification is sent.

Following Michel's comment: https://github.com/airbytehq/airbyte/pull/2706#discussion_r608334608 we may want to add that in the env files?

Additionally, some users also get confused with the warning:
Screenshot 2021-04-08 at 19 44 16

The `API_URL` and similarly the new `WEBAPP_URL` should be defined in the .env files otherwise it would print that warning in the console when starting the server up?

## How
Define the variables in the .env files but make sure they have appropriate fallback default value in envConfigs anyway (like the other variables)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -17,3 +17,5 @@ TRACKING_STRATEGY=segment
 # already exist on the host filesystem and MUST be parents of *_ROOT.
 # Issue: https://github.com/airbytehq/airbyte/issues/577
 HACK_LOCAL_ROOT_PARENT=/tmp
+WEBAPP_URL=http://localhost:8000/
```
Comment (jrhizor) [inline .env:20]: I think these should have the same format:

Either:
```
WEBAPP_URL=http://localhost:8000/
API_URL=http://localhost:8001/api/v1/
```
or
```
WEBAPP_URL=http://localhost:8000
API_URL=http://localhost:8001/api/v1
```

Comment (jrhizor) [reply .env:20]: Usage probably needs to be updated for one or the other.

**Review by jrhizor — APPROVED** [review]
I like this. 

As an aside, I think all of our constants should be defined in .env.

File: docker-compose.yaml [hunk 2]
```diff
@@ -80,7 +80,7 @@ services:
     environment:
       - AIRBYTE_ROLE=${AIRBYTE_ROLE:-}
       - AIRBYTE_VERSION=${VERSION}
-      - API_URL=${API_URL}
+      - API_URL=${API_URL:-}
```
Comment (michel-tricot) [inline docker-compose.yaml:83]: I am not a fan of having a the default hardcoded in the code. I would prefer if it failed

**Pull request merged**